ambry-clustermap/src/main/java/com/github/ambry/clustermap/HelixBootstrapUpgradeUtil.java

```diff
@@ -897,73 +901,50 @@ public void migrateToFullAuto(String commaSeparatedResources, WagedHelixConfig w
             }
             // b. Update instance config for each instance
             for (String instance : instances) {
-              setupInstanceConfig(dcName, instance, resource, configAccessor, propertyStoreAdapter);
+              setInstanceConfig(dcName, instance, resource, configAccessor, propertyStoreAdapter);
             }
-            allInstances.addAll(instances);
-          }
-
-          // 3. No resource config to update for now
-
-          // 4. Validate configs
-          // 4.1 Instance validation
-          Map<String, Boolean> instanceValidationResultMap =
-              helixAdmin.validateInstancesForWagedRebalance(clusterName, new ArrayList<>(allInstances));
-          Set<String> invalidInstances = new HashSet<>();
-          for (Map.Entry<String, Boolean> entry : instanceValidationResultMap.entrySet()) {
-            if (entry.getValue() == Boolean.FALSE) {
-              invalidInstances.add(entry.getKey());
-            }
-          }
-          if (!invalidInstances.isEmpty()) {
-            logger.error("Validation of waged rebalancer configuration failed for instances {}", invalidInstances);
-            throw new IllegalStateException(
-                "Instances" + invalidInstances + "are not configured correctly for waged rebalancer");
-          }
-          // 4.2 Resource validation
-          Map<String, Boolean> resourceValidationResultMap =
-              helixAdmin.validateResourcesForWagedRebalance(clusterName, new ArrayList<>(resources));
```

Comment:
Removed validation for now since `ZkHelixAdmin.validateResourcesForWagedRebalance` APIs don't seem to support tags and expect all the instances of the cluster to be configured for Waged rebalancer. We can probably do manual validation via curl if that works.